subprojects/docs/src/samples/customPlugin/plugin/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
 // END SNIPPET local-groovy-dependencies
 // END SNIPPET use-plugin
 
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
```

Comment:
Should this be using the `plugins {}` block? I'm assuming this isn't in the user manual, but I think all samples should use recommended syntax where possible if it's cheap to change.